src/sysen2/mldev.108

```diff
@@ -55,6 +55,9 @@ FILBLN:	0	;FILE LENGTH, IN BYTES OF SIZE WRITTEN IN.
 FILBSZ:	0	;BYTE SIZE FILE WRITTEN IN.
 FILDTP:	0	;-1 => WE KNOW THE CREATION DATE OF THIS FILE.
 FILDAT:	0	;FILE CREATION DATE (IF KNOWN).
+3char:  0	;-1 if hostname is 3 characters long, else 2 characters long
```

Comment:
I see `3char` is set to 0 or -1, but I don't see that the value is checked later.